packages/builder/src/components/design/settings/controls/FormFieldSelect.svelte

```diff
@@ -25,16 +25,21 @@
   const getOptions = (schema, type) => {
     let entries = Object.entries(schema ?? {})
     let types = []
-    if (type === "field/options" || type === "field/barcode/qr") {
+    if (type === "field/options") {
       // allow options to be used on both options and string fields
       types = [type, "field/string"]
     } else {
       types = [type]
     }
 
-    types = types.map(type => type.slice(type.indexOf("/") + 1))
+    types = types.map(type => {
+      let fieldTypeRaw = type.slice(type.indexOf("/") + 1)
+      let fieldTypeParsed = fieldTypeRaw.replace("/", "")
+      return fieldTypeParsed
```

Comment:
Was the problem here that the field type was `field/barcode/qr` but the actual datatype is `barcodeqr`?

Reply:
The reason I ask is just that if it is, we could actually just update the manifest to change the setting type to `field/barcodeqr` couldn't we? Which would just be a one line change and not require and other changes (I think).

Reply:
That's essentially it. My only concern in modifying the type was any potential side-effect that I might miss. I've run through some more testing and it does seem a manifest update is all that's required here.